dc-cudami-editor/src/locales/en/translation.json

```diff
@@ -26,6 +26,14 @@
     "createTopic": "Create a new topic",
     "createWebpage": "Create a new webpage",
     "createWebsite": "Create a new website",
+    "datePickerAriaLabel": {
+      "calendarAriaLabel": "Toggle calendar",
```

Comment:
```suggestion
      "toggleCalender": "Toggle calendar",
```

Reply:
Motivation für die Benamung der Schlüssel hier: diese sollen identisch zu den Namen der DatePicker Komponente sein, siehe [hier](https://github.com/wojtekmaj/react-date-picker#props).

@jbaiter Soll ich diese Änderung dann trotzdem machen, wie von @morpheus-87 vorgeschlagen 🤔

Reply:
Warum müssen die identisch sein? `calendarAriaLabel` ist ja nur der Parameter-Name, die Variable die du damit übergibst kann ja ruhig etwas sprechender sein. Würde das auch in `toggleCalendar` umbenennen.